plugins/Marketplace/vue/src/InstallAllPaidPluginsButton/InstallAllPaidPluginsButton.vue

```diff
@@ -0,0 +1,91 @@
+<!--
+  Matomo - free/libre analytics platform
+
+  @link    https://matomo.org
+  @license https://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+
+<template>
+  <button
+    class="btn"
+    @click.prevent="onInstallAllPaidPlugins()"
+    :disabled="disabled"
```

Comment:
Should we move some CSS around to have a better readability when the button is disabled?

For `.cta-container .btn.disabled` there is `color: #858585` set ([source](https://github.com/matomo-org/matomo/blob/e52dc914d827648048838333a1826ee1aa67e7db/plugins/Marketplace/stylesheets/marketplace.less#L323)), to not have white on light grey, but grey on light grey.

Reply:
can do